extensions/aproc/proc/ingest/drivers/impl/digitalglobe.py

```diff
@@ -159,14 +166,19 @@ def to_item(self, url: str, assets: list[Asset]) -> Item:
         )
         if eo__cloud_cover != -999000.0:
             item.properties.eo__cloud_cover = eo__cloud_cover
+
+        if xml_path != self.xml_path:
+            os.remove(xml_path)
+        if tif_path != self.tif_path:
+            os.remove(tif_path)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0